src/transformers/processing_utils.py

```diff
@@ -463,7 +463,7 @@ def from_pretrained(
         try:
             processor_dict, kwargs = cls.get_processor_dict(pretrained_model_name_or_path, **kwargs)
         except EnvironmentError as e:
-            if "does not appear to have a file named processor_config.json." in str(e):
+            if "a file named processor_config.json" in str(e):
```

Comment:
Not the best solution. I can go for another approach that uses `_raise_exceptions_for_missing_entries` when calling `cached_file` inside `get_processor_dict`.

Reply:
Yes please!

Reply:
Done